src/nebari_plugin_self_registration/terraform/modules/self-registration/chart/templates/config-map.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-config
+data:
+  config.yaml: |-
+    {{ toYaml .Values.app_configuration | nindent 4 }}
```

Comment:
If this is going to contain the client credentials for Keycloak, should be a secret instead of a configmap

Reply:
switched to secret